agent/adapter.cpp

```diff
@@ -539,7 +541,7 @@ void Adapter::thread()
 
 		// Try to reconnect every 10 seconds
 		g_logger << LINFO << "Will try to reconnect in " << m_reconnectInterval << " milliseconds";
-		dlib::sleep(m_reconnectInterval);
+		this_thread::sleep_for(chrono::milliseconds(m_reconnectInterval));
```

Comment:
In future commits, m_reconnectInterval could become a 'std::chrono::duration' type. This would alleviate the need to create one in multiple places. It would also allow time periods to be added/subtracted easily.